geoportailv3/views/luxprintproxy.py

```diff
@@ -78,9 +79,23 @@ class LuxPrintProxy(PrintProxy):
     @view_config(route_name="lux_printproxy_report_create")
     def lux_report_create(self):
         token = self.config["authtkt_secret"]
-        print_urls = self.config["print_urls"]
+        print_servers = DBSession.query(LuxPrintServers).all()
+        print_urls = [print_server.url for print_server in print_servers]
+        urllib2.getproxies = lambda: {}
```

Comment:
why this ?

Reply:
to force urllib to not use the proxy to communicate with print although it is configured in wsgi.
until now we used the no_proxy variable, but as the hostnames are volatile, we cannot save them in the config file